wpilibc/src/main/native/include/frc/simulation/DifferentialDrivetrainSim.h

```diff
@@ -117,6 +119,38 @@ class DifferentialDrivetrainSim {
    */
   Pose2d GetPose() const;
 
+  /**
+   * Get the right encoder distance in meters.
+   * @return The encoder distance.
+   */
+  units::meter_t GetRightDistance() {
+    return units::meter_t{GetOutput(State::kRightPosition)};
+  }
```

Comment:
The API isn't consistent on position vs distance. Also, this should be const-qualified.